e2e/cypress/integration/archived_channel/archived_leave_channel_spec.js

```diff
@@ -270,6 +270,36 @@ describe('Leave an archived channel', () => {
 
         // * Private archived channels you are not a member above are not available on channel switcher
         cy.contains('#suggestionList', otherChannelName).should('not.exist');
+        cy.get('#quickSwitchModalLabel button.close').click();
+    });
+    it('MM-T1677 Archived channels are not shown as unread in channel switcher', () => {
+        // # As the test user join a public channel then open any other channel in the drawer
+        cy.uiCreateChannel('arhived-not-read').then(({name}) => {
```

Comment:
Should this be below to match with the one when [switching a channel](https://github.com/mattermost/mattermost-webapp/pull/6511/files#diff-6c654b6a642cee5dbbe696d173d8db2cR296)?
```suggestion
        cy.uiCreateChannel('archived-not-read').then(({name}) => {
```